orchestrator/consensus/handler.go

```diff
@@ -1,111 +1,179 @@
 package consensus
 
 import (
-	"fmt"
+	"github.com/lukso-network/lukso-orchestrator/orchestrator/utils"
 
 	"github.com/ethereum/go-ethereum/common"
 	eth1Types "github.com/ethereum/go-ethereum/core/types"
 	"github.com/lukso-network/lukso-orchestrator/shared/types"
 )
 
-// processPandoraHeader
+// processPandoraHeader method process incoming pandora shard header from pandora chain
```

Comment:
The first sentence of a comment should be a one-sentence summary that starts with the name being declared.
```suggestion
// processPandoraHeader processes pandora shard header coming from pandora chain
```